<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportProjectWidget</class>
 <widget class="QWidget" name="ImportProjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>534</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbProject">
     <property name="title">
      <string>Project</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lFileName">
        <property name="text">
         <string>File name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bOpen">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string> Select the file to import</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbObjects">
     <property name="title">
      <string>Objects to Import</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeView" name="tvPreview"/>
      </item>
      <item>
       <widget class="QCheckBox" name="chbUnusedObjects">
        <property name="text">
         <string>Import unused object</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbImportTo">
     <property name="title">
      <string>Import to</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lFolder">
        <property name="text">
         <string>Folder:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
